datasets/modeled/industry/dimension_mappings/subsector_to_subsector.csv

```diff
@@ -0,0 +1,15 @@
+from_id,to_id
+cement_concrete,cement_concrete
+chemical,chemical
+chemical_other,chemical
```

Comment:
I wonder what this distinction is and whether we want to keep it at the project level ... @sjsmith11?